content/articles/setting-name-servers.md

```diff
@@ -58,16 +58,20 @@ DNSimple's listing of NS records for the domain will be updated to match the nam
 
 Pointing the name servers to DNSimple provider will cause the domain to resolve using the DNS records configured in your DNSimple account.
 
-To change the name servers to DNSimple, follow the steps in the previous section, and use the [DNSimple name servers](/articles/dnsimple-nameservers).
+To change the name servers to DNSimple, follow the steps in [Delegating a Domain registered with DNSimple to DNSimple](/articles/delegating-dnsimple-registered).
 
 ## Glue records
 
-If you are adding a name server that is a child of the domain, glue records are required. In this case, you will be prompted to add the glue upon clicking <label>Change Name Servers</label>.
+If you are adding a name server that is a child of the domain, glue records are required. In this case, you will be prompted about the missing glue records upon clicking <label>Change Name Servers</label>. The glue records can be added by creating a [name server set](/articles/name-server-sets/#creating-an-account-name-server-set) and clicking on <label>Add a name server set</label> to apply it on the <label>Edit delegation</label> page.
 
 <div class="mb4 aspect-ratio aspect-ratio--16x9 z-0">
   <iframe src="https://www.youtube.com/embed/m_RaPIRNxFs?rel=0&modestbranding=1&cc_load_policy=1&cc_lang_pref=en" class="aspect-ratio--object" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen=""></iframe>
 </div>
 
+<info>
+The above video also demonstrates adding missing glue records directly from the Edit delegation page. However, this is no longer supported. Glue records can only be added via applying [name server sets](/articles/name-server-sets/#creating-an-account-name-server-set) that contain the necessary glue records.
```

Comment:
I wonder if we need to remove the video completely (or record a new one) 🤔

Reply:
I think the video is really well-made and most of the video still contains useful information. Perhaps we can leave it up for now and see if this clarifying note is sufficient?

Reply:
I agree that we should leave the video for now.

Reply:
I also agree